src/main/resources/frontend/src/components/AboutPage.vue

```diff
@@ -91,6 +91,11 @@ onUnmounted(() => {
           <p class="tenure">Jan 2024-Dec 2025</p>
           <p class="contributions">Developed class-requirement enforcement systems, like late-days and git-commits</p>
         </div>
+        <div class="person">
```

Comment:
I'm kind of surprised we haven't already created a component to encapsulate the details of creating a new person. 

I'm imagining something like the following:
```vue
<Person
    name="Dallin Webeke"
    url="awesome.com"
    fa-icon="unicorn"
    tenure="Since forever ago"
    contributions="Robust Vue Management"
></Person>
```
or, a for loop that displays all the persons based on an in-memory (potentially dynamically loaded) list of contributors.

That doesn't need to be incorporated in time for this PR, but you can create a mental note (or an issue) if you agree.